api/database/models.py

```diff
@@ -55,6 +55,8 @@ class Prediction(Base):
     Vorkath_bot = Column(Float)
     Barrows_bot = Column(Float)
     Herblore_bot = Column(Float)
+    Unknown_bot = Column(Float)
+    Clue_Scroll_bot = Column(Float)
```

Comment:
Isn't this already on line 45?

Reply:
resolved in [push](https://github.com/Bot-detector/Bot-Detector-Core-Files/pull/426/commits/20220e281640b2241d2e992a7d8c0a08e953c936)